content/en/serverless/installation/java.md

```diff
@@ -85,11 +81,66 @@ datadog-ci lambda instrument -f my-function -f another-function -r us-east-1 -e
 
 More information and additional parameters can be found in the [Datadog Serverless CLI][2].
 
+### Install the Datadog tracing client
+
+[Configure the layers][3] for your Lambda function using the ARN in the following format:
+
+```
+arn:aws:lambda:<AWS_REGION>:464622532012:layer:dd-trace-java:<VERSION>
+```
+
+The latest `VERSION` is {{< latest-lambda-layer-version layer="dd-trace-java" >}}.
+
 [1]: https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
 [2]: https://docs.datadoghq.com/serverless/serverless_integrations/cli
+[3]: https://docs.aws.amazon.com/lambda/latest/dg/invocation-layers.html
+{{% /tab %}}
+{{% tab "Serverless Framework" %}}
+
+The [Datadog Serverless Plugin][1] automatically adds the Datadog Lambda Library to your functions using Lambda Layers, and configures your functions to send metrics, traces, and logs to Datadog through the [Datadog Lambda Extension][2].
```

Comment:
```suggestion
The [Datadog Serverless Plugin][1] automatically configures your functions to send metrics, traces, and logs to Datadog through the [Datadog Lambda Extension][2].
```

The Serverless Framework plugin can't actually add the Library as a layer because Java is a compiled language.